docs/settings.rst

```diff
@@ -9,11 +9,12 @@ line::
 If you used the ``pelican-quickstart`` command, your primary settings file will
 be named ``pelicanconf.py`` by default.
 
-You can also specify extra settings via ``-e`` / ``--extra-settings`` option
-flags, which will override default settings as well as any defined within
-settings files::
+You can also specify settings via ``-e`` / ``--extra-settings`` option
+flags. It will override default settings as well as any defined within the
+setting file. Note that values must follow JSON notation::
+
+    pelican content -e SITENAME='"A site"' READERS='{"html": null}' DELETE_OUTPUT_DIRECTORY=true
```

Comment:
I had imagined that each specified setting would need to be prefixed with `-e`, such as:
```
pelican content -e SITENAME='"A site"' -e READERS='{"html": null}' -e DELETE_OUTPUT_DIRECTORY=true
```

Is that not necessary with this implementation? If multiple `-e` prefixes are not necessary, can other option flags be added _after_ multiple `--extra-settings` parameters, such as the following?
```
pelican content -e SITENAME='"A site"' READERS='{"html": null}' -o /tmp/output -t my-theme --autoreload
```

Reply:
Also, since Pelican already has a `-d` / `--delete-output-directory` option flag, perhaps we should replace `DELETE_OUTPUT_DIRECTORY=true` with another example of a settings override? It seems to me that particular example doesn't make much sense when there is a built-in option flag already available.

Reply:
> Is that not necessary with this implementation?

It's not necessary but it has nothing to do with this PR. This behaviour is determined by `argparse`'s `nargs='*'` (https://docs.python.org/3/library/argparse.html#nargs). 

> can other option flags be added after multiple --extra-settings parameters

They can. 

> perhaps we should replace DELETE_OUTPUT_DIRECTORY=true with another example of a settings override?

Agree. I replaced it with `CACHE_CONTENT=true`.

Reply:
Got it. Many thanks for the explanation!